cmake/Modules/FindSDL2_ttf.cmake

```diff
@@ -33,7 +33,7 @@ find_path(SDL2_TTF_INCLUDE_DIR SDL_ttf.h
   HINTS
     ENV SDLTTFDIR
     ENV SDLDIR
-  PATH_SUFFIXES include/SDL2 include
+    PATH_SUFFIXES include/SDL2 include headers/SDL2 headers
```

Comment:
SDL2 should be enough

Reply:
You re surely right for the two points above but this particular point it did not work out of the box for me, is to avoid passing flags/env vars as much as possible.

Reply:
headers folder always searched by cmake, and it will use the correct arch settings, if you run setarch. Also in haikuporter with correct prefix (_x86), so SDL header files can be found with only the SDL2 hint added here. See other ports for reference:
https://github.com/haikuports/haikuports/search?q=PATH_SUFFIXES+SDL&unscoped_q=PATH_SUFFIXES+SDL

Reply:
true you are correct I ve tried locally seems working finally.